src/App.js

```diff
@@ -14,23 +14,25 @@ import './App.css';
 
 const App = () => {
   return (
-    <MapCenter>
-      <Locations>
-        <Router>
-          <div className="app">
-            <main className="app__content">
-              <Switch>
-                <Route path="/" exact component={MapPage} />
-                <Route path="/list" component={ListPage} />
-                <Route path="/settings" component={Settings} />
-                <Route path="/help" component={Help} />
-              </Switch>
-            </main>
-            <Footer />
-          </div>
-        </Router>
-      </Locations>
-    </MapCenter>
+    <GoogleMaps>
+      <MapCenter>
+        <Locations>
```

Comment:
As long as we’re renaming context providers, I’d suggest renaming these to `MapCenterProvider` and `LocationsProvider`, respectively. ← Alternately, this could be done in a separate PR to isolate those changes

Reply:
Good suggestions 👍 ! We are going to create a new Jira ticket to fix it in the future 😄.

Reply:
[Jira Ticket](https://the-collab-lab.atlassian.net/browse/TCL30-49)  ^^ 😄